packages/app-content-pages/server/server.js

```diff
@@ -22,3 +27,18 @@ app.prepare().then(() => {
     console.log(`> Ready on http://localhost:${port}`)
   })
 })
+
+function setCacheHeaders (req, res) {
+  let maxAge = 60 // 1 minute
```

Comment:
Do you want these to be configured via ENV vars with a default fall back? I.e. change a deploy value instead of code to get changes into play?